package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "playcanvas",
+  "version": "1.0.0",
+  "description": "JavaScript game engine built on WebGL and WebVR. https://playcanvas.com/",
+  "main": "build/output/playcanvas-latest.min.js",
+  "directories": {
+    "example": "examples",
+    "test": "tests"
+  },
+  "scripts": {
+    "build": "cd build && node build.js -l 1 -o output/playcanvas-latest.min.js && node build.js -l 0 -o output/playcanvas-latest.js && node build.js -l 0 -d -o output/playcanvas-latest.dbg.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/playcanvas/engine"
+  },
+  "engines": {
+      "node": ">= 0.6.12"
+  },
+  "author": "PlayCanvas <support@playcanvas.com>",
```

Comment:
Please move keys for better readability to such order:
name, version, author, homepage, description, license, main, bugs, repository, dependencies, devDependencies, directories, engines, scripts